bins/revme/src/cmd/evmrunner.rs

```diff
@@ -125,3 +123,36 @@ impl Cmd {
         Ok(())
     }
 }
+
+fn run_benchmark(bytecode_str: Cow<str>) {
+    let evm = Evm::builder()
+        .with_db(BenchmarkDB::new_bytecode(Bytecode::new()))
+        .modify_tx_env(|tx| {
+            tx.caller = address!("1000000000000000000000000000000000000000");
+            tx.transact_to = TxKind::Call(address!("0000000000000000000000000000000000000000"));
+        })
+        .build();
+
+    let host = DummyHost::new(*evm.context.evm.env.clone());
+    let instruction_table = make_instruction_table::<DummyHost, LatestSpec>();
+    let contract = Contract {
+        input: Bytes::from(hex::decode("").unwrap()),
+        bytecode: to_analysed(Bytecode::new_raw(
+            hex::decode(bytecode_str.to_string()).unwrap().into(),
+        )),
+        ..Default::default()
+    };
+    let mut criterion = criterion::Criterion::default();
+
+    let mut criterion_group = criterion.benchmark_group("revme");
+    criterion_group.bench_function("bytecode", |b| {
+        b.iter(|| {
+            Interpreter::new(contract.clone(), u64::MAX, false).run(
+                EMPTY_SHARED_MEMORY,
```

Comment:
```suggestion
                SharedMemory::new(); 
```

Shared memory is preallocated and shared between multiple calls, so having SharedMemory::new() will have an initialization hit but it will be slightly faster on memory expansion.